.yamllint.yaml

```diff
@@ -3,6 +3,7 @@ extends: default
 ignore: |
   .clang-tidy
   *.param.yaml
+  codecov.yaml
```

Comment:
Why do you need to ignore this?

Reply:
I got the following errors from yamllint, but CodeCov yaml validator says it is valid (see above comment).
So I ignore yamllint.
Do you have any idea to fix it?

```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

codecov.yaml
  2:23      error    truthy value should be one of [false, true]  (truthy)
  7:10      error    string value is redundantly quoted with any quotes  (quoted-strings)
  12:20     error    truthy value should be one of [false, true]  (truthy)
  13:13     error    truthy value should be one of [false, true]  (truthy)
  14:15     error    truthy value should be one of [false, true]  (truthy)
  15:14     error    truthy value should be one of [false, true]  (truthy)
  18:11     error    string value is redundantly quoted with any quotes  (quoted-strings)
  20:20     error    truthy value should be one of [false, true]  (truthy)
```

Reply:
I set this rule to keep the consistent style.
Please follow the rule if there are no special reasons.
If not, there will be many valid styles and we'll get confused.

Reply:
I see, I did not know that yes and no in CodeCov yaml can be substituted with true and false.
Now I've passed yamlint.

Reply:
d119c887